lib/Bio/KBase/NarrativeMethodStore/Client.pm

```diff
@@ -913,6 +913,7 @@ MethodParameter is a reference to a hash where the following keys are defined:
 	disabled has a value which is a NarrativeMethodStore.boolean
```

Comment:
These files are all generated:
Client.pm
client.py
DropdownOptions.java
DynamicDropdownOptions.java
MethodParameter.java

Reply:
Those two client files could be deleted at some point since only the java files are actually used.

Reply:
The perl client maybe, since perl isn't widely used, especially for infrastructure. However it's very convenient to be able to just clone the repo and use the python client. The workspace repo is the same way, and I use that client all the time, although it's not actually used in the workspace code.